src/compiler/program.ts

```diff
@@ -4229,6 +4229,12 @@ export function createProgram(rootNamesOrOptions: readonly string[] | CreateProg
             }
         }
 
+        if (options.isolatedDeclarations) {

```

Comment:
Does `isolatedDeclarations` means its "declaration: true` if yes we need to update `createComputedCompilerOptions` to handle that... otherwise we need error here if this is specified with declarations

Reply:
Ok. Will add the error.